src/popup.js

```diff
@@ -0,0 +1,39 @@
+const popup = (el) => {
+  /* eslint-disable */
+  const popdiv = document.querySelector('.popu');
```

Comment:
- [ ] Kindly enable the linter for this file and remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
Addressed 👍🏼